sim/src/main/makefrag/midasexamples/Makefrag

```diff
@@ -30,15 +30,12 @@ else
 	firesim_sbt_project := {file:${firesim_base_dir}/}firesim
 endif
 
-submodules = \
-    $(addprefix $(firesim_base_dir)/,. midas midas/targetutils firesim-lib) \
-    $(addprefix $(chipyard_dir)/, \
-		$(addprefix generators/, rocket-chip) \
-		$(addprefix tools/, firrtl chisel3))
+chisel_src_dirs = \
+		$(addprefix $(firesim_base_dir)/,. midas midas/targetutils firesim-lib) \
+		$(addprefix $(chipyard_dir)/generators/, chipyard rocket-chip/src, rocket-chip/api-config-chipsalliance)
```

Comment:
Is there a point to being so specific with the wildcarding? Do we not want to recompile if, say, FIRRTL changes?

Reply:
there's less point now after the SUFFIX rule change, but firrtl changes should be caught by the FIRRTL.jar prerequisite.